src/Action/AddAction.php

```diff
@@ -127,9 +128,8 @@ protected function _post(): ?Response
             return $this->_stopped($subject);
         }
 
-        $saveCallback = [$this->_model(), $subject->saveMethod];
         /** @phpstan-ignore argument.type */
-        if (call_user_func($saveCallback, $subject->entity, $subject->saveOptions)) {
+        if (call_user_func([$this->_model(), $this->saveMethod()], $entity, $this->saveOptions())) {
```

Comment:
This is just wrong. The whole point of setting the entity, saveMethod and saveOptions to the subject is to allow using customizing them using the `Crud.beforeSave` event. Hence those values should be used from the subject after the event execution.

Reply:
Then edit must be wrong, as those are doing different ways of handling this.